astropy/units/tests/test_format.py

```diff
@@ -815,18 +815,53 @@ def test_fits_scale_factor_errors():
     assert x.to_string(format="fits") == "10**2 erg"
 
 
-def test_double_superscript():
+@pytest.mark.parametrize(
```

Comment:
I agree with @pllim that this does not make things clearer. How about parametrizing as.
```
"unit, latex, unicode",
[
    (u.deg, r"$\mathrm{{}^{\circ}}$", "°"),
    (u.deg**2, ...)
]
```
and then obviously,
```
assert unit.to_string("latex") == latex
assert unit.to_string("unicode") == unicode
```

That way we keep clear what expected string belongs to what unit.